lldb/test/API/tools/lldb-dap/console/TestDAP_console.py

```diff
@@ -104,3 +127,27 @@ def test_empty_escape_prefix(self):
             "Help can be invoked",
             command_escape_prefix="",
         )
+
+    @skipIfWindows
+    @skipIfRemote
+    def test_exit_status_message(self):
+        source = "main.cpp"
+        program = self.getBuildArtifact("a.out")
+        self.build_and_launch(program, commandEscapePrefix="")
+        breakpoint1_line = line_number(source, "// breakpoint 1")
+        breakpoint_ids = self.set_source_breakpoints(source, [breakpoint1_line])
+        self.continue_to_breakpoints(breakpoint_ids)
+
+        # Kill lldb-server process.
+        process_name = "lldb-server"
```

Comment:
this is error prone because multiple lldb-servers might be running at the same time. You need the lldb-server or debugserver that is child of the current lldb-vscode process and not the most recent one